content/en/apps/guides/data/analytics/building-dbt-models.md

```diff
@@ -0,0 +1,376 @@
+Forms may be specific to each CHT application; additional models will need to be developed to analyze data from responses to these custom forms.
+One additional model will be needed for each form, and for any aggregations, dashboards, or reusable views that use those form responses as input.
+If using the [configurable contact hierarchy]({{< ref "apps/reference/app-settings/hierarchy#app_settingsjson-contact_types" >}}), it may also be useful to add models for other contact types.
+
+## Setup
+
+To create application specific models, create a  [new dbt project](https://docs.getdbt.com/reference/commands/init).   Edit the `packages.yml` in your new dbt project to add the [CHT Pipeline](https://docs.getdbt.com/docs/build/packages)  as a dependency.   So that you can track changes in your models, put your dbt new project in a GitHub repository (it may be public or private).
+```yml
+packages:
+  - git: "https://github.com/medic/cht-pipeline"
+    revision: "v1.0.0"
+```
+To avoid breaking changes in downstream models, include a version tag in the dependency.
+
+In CHT Sync config, set the URL of dbt GitHub repository to the `CHT_PIPELINE_BRANCH_URL` [environment variable]({{< relref "apps/guides/data/analytics/environment-variables" >}}), either in ´.env´ if using ´docker compose´, or in ´values.yaml´ if using Kubernetes.
+
+### Deploying models
+
+CHT Sync automatically checks for updates to the dbt project at `CHT_PIPELINE_BRANCH_URL`
+Use the main branch for changes that should be released; they will applied as soon as they are pushed to the repository.
+For models that are in development, any other branch can be used.
+
+Rebuilding tables can take some time (a rough estimate is several hours for tables between 1M and 10M rows, upt to 24H for tables between 10M and 100M rows) so plan for affected dashboards to be out of date when releasing new changes to models; only changed tables and their dependencies will be rebuilt.
+
+New releases of the base models are rare, but schema changes to the CHT or new features may require changes.
+Generally, these changes will be backwards compatible so that a new release of the CHT, or to the base models, does not break application specific models.
+
+When it is necessary to update the base models, update the version tag in the dependency that refers to cht-pipeline, make any changes to application specific models that are necessary, rerun any unit tests, and push to the main branch to release. 
+
+
+### Testing models and dashboards
+
+It is highly encouraged to write dbt [tests](https://docs.getdbt.com/docs/build/data-tests) for application specific models to ensure that they are accurate and to avoid releasing broken models. Examples can be found in [CHT Pipeline](https://github.com/medic/cht-pipeline/tree/main/test).
```

Comment:
```suggestion
It is highly encouraged to write [dbt tests]({{< ref "apps/guides/data/analytics/testing-dbt-models" >}}) for application-specific models to ensure that they are accurate and to avoid releasing broken models. Examples can be found in the [cht-pipeline repository](https://github.com/medic/cht-pipeline/tree/main/test).
```

Reply:
I linked the guide @lorerod put together about dbt tests.